sklearn/preprocessing/tests/test_discretization.py

```diff
@@ -129,23 +143,21 @@ def test_ignored_invalid():
         assert_raises(ValueError, est.fit, X)
 
 
-def test_same_min_max():
+@pytest.mark.parametrize('strategy', ['uniform', 'kmeans', 'quantile'])
+@pytest.mark.parametrize('n_bins', [2, 3])
+def test_same_min_max(strategy, n_bins):
```

Comment:
Do we test ``n_bins_[0] = 1`` somewhere? Or do you think we don't need such a test?
Also, maybe we don't need to test multiple ``n_bins`` here?

Reply:
`n_bins = 1` is tested in arrays (`test_invalid_n_bins_array`) and as an integer (`test_invalid_n_bins`).

Reply:
Oh I misread your comment, I'll add a line to test `n_bins_`.